tokenserver/tests/assignment/test_sqlnode.py

```diff
@@ -29,6 +29,8 @@ def setUp(self):
         self.backend.add_service('sync-1.5', '{node}/1.5/{uid}')
         self.backend.add_service('queuey-1.0', '{node}/{service}/{uid}')
         self.backend.add_node('sync-1.0', 'https://phx12', 100)
+        self.backend.add_node('sync-1.5', 'https://phx11', 100)
+        self.backend.migrate_new_user_percentage = 0
```

Comment:
If I delete these two lines, all of the tests still seem to pass. Are they necessary, perhaps only under some test configurations?

Reply:
probably in there for completion sake, but yeah, not really needed.